### DISCUSSION TITLE
[RIS-Import] Error during processing of title fields T1 or TI during RIS import

### DISCUSSION THREAD

**Discussion by felixhorch** [asreview/asreview#1153] [Troubleshooting (report errors in the "Issues" tracker)]
Hello!

When processing a .RIS file exported from Citavi, I noticed an error when processing the `T1` or `TI` title fields: Apparently, the title of the publications is only recognized correctly by ASReview if the title field is labeled `TI`. But if the title is exported as `T1`, as it is apparently implemented in Citavi, ASReview does not show any titles, neither in the search nor in the review. However, as far as I understand the RIS specification, both are allowed and correct. 

Couldn't this be added relatively easily in the RIS processing?

Best regards

Felix

**Comment by Rensvandeschoot** [comment]
See issue https://github.com/asreview/asreview/issues/1154
